homepage/tests/post_review_tests.py

```diff
@@ -21,46 +21,64 @@ def review_details():
 @pytest.mark.django_db
 def test_post_invalid_review(invalid_review_details):
     invalid = False
-    form = ReviewForm(data=invalid_review_details)
-
+    form = ReviewForm(data=invalid_review_details, course=invalid_review_details.get('course'))
     try:
         form.save()
     except ValueError:
         invalid = True
-
     assert invalid
 
 
 @pytest.mark.django_db
 def test_post_valid_review(review_details):
-    form = ReviewForm(data=review_details)
-
+    form = ReviewForm(data=review_details, course=review_details.get('course'))
     if form.is_valid():
         review = form.save()
         assert Review.objects.filter(pk=review.id).exists()
     else:
         assert False
 
 
-# --------Front End testing-------- #
+@pytest.mark.parametrize("course_id, prof_id_list", [
+    ((10221), [1]),  # Grammatica in Arithmancy, Septima Vector

```

Comment:
please try to remove the parenthesis 
```suggestion
    (10221, [1]),  # Grammatica in Arithmancy, Septima Vector
```
same for the lines below